packages/cypress/src/pageObjects/Lookup/examples.mdx

```diff
@@ -0,0 +1,30 @@
+-   ##### How to type something in the Lookup
+
+```js
+import { Lookup } from '@rainbow-modules/cypress/pageObjects';
+
+const lookup = new Lookup('#lookup-selector');
+lookup.input.type('rainbow');
+```
+
+-   ##### How to clear the Lookup value
+
+```js
+import { Lookup } from '@rainbow-modules/cypress/pageObjects';
+
+const lookup = new Lookup('#lookup-selector');
+lookup.clearButton.click();
+```
+
+-   ##### How to pick an option
+
+```js
+import { Lookup } from '@rainbow-modules/cypress/pageObjects';
+
+const lookup = new Lookup('#lookup-selector');
+lookup.input.click().then(($input) => {
+    const ariaControls = $input.attr('aria-controls');
+    const dropdown = new InternalDropdown(`#${ariaControls}`);
+    dropdown.options[0].click();
```

Comment:
here the user should not know about InternalDropdown, we need to define another way to select the options

Reply:
we need to do something like:
```
lookup.options[0].click()
```